shared-articles/what does dcdd do.md

```diff
@@ -0,0 +1,90 @@
+
+Therefore, since the network participants are each following the same set of rules, and everyone is sharing the same ledger between each other, it's impossible to go back and retroactively alter the ledger. You can only add new information to the ledger. That's what makes old blocks immutable.
+
+The software that makes this all possible on the Decred blockchain is dcrd. dcrd is written in Go (golang), and implements all the consensus rules and core blockchain ledger functionality. It is important to note that dcrd does NOT include wallet functionality, and was designed that way for security reasons. You can consider dcrd to be your personal Decred blockchain server, which would connect you to the rest of the Decred network.
+
+## dcrd responsibilities:
+dcrd is the backbone of Decred's peer-to-peer network. dcrd acts as a 'full node', short for 'fully-validating node', which means that it fully validates all transactions and blocks, as opposed to trusting a 3rd party or someone else's ledger. In summary, a dcrd full node:
+
+* Fully validates all transactions and blocks (by enforcing the Decred consensus rules),
+* Maintains the entire past transactional ledger of Decred,
+* Relays newly mined blocks,
+* Gives the ability to mine blocks,
+* Allows relaying of transactions to other Decred nodes around the world (by sharing a transaction mempool),
+* Allows sharing a list of peers in the Decred network so that each new peer can talk to as many other peers as possible.
+
+In terms of benefits, since dcrd fully validates every block and transaction, it provides the highest security and privacy possible when used with a wallet that’s also in full validation mode. This is ideal for individuals, businesses, and services that need the most reliable and accurate data about transactions.
+
+## Full Node Distinctions
+As Decred builds out its infrastructure, it becomes apparent that the underlying security is a high priority. Each time a new service is added, it increases the opportunity for node participation. This is done on multiple levels, and it’s important to note that not every node is equal. There are two factors to consider when determining the usefulness of a node:
+
+**Uptime**
+* High-uptime (running 24/7/365, or close to it)
+* Low-uptime (running intermittently)
+**With inbound connections**
+* No (default)
+* Yes (requires the user to open up a port in their firewall)
+
+**Dcrd configuration options include**
+* High-uptime with inbound connections
+* High-uptime without inbound connections
+* Low-uptime with inbound connections
+* Low-uptime without inbound connections
+
+The Full node that supports the network the best is the High-uptime with inbound connections.
```

Comment:
Suggested change:
`The full node that supports the network the best has high-uptime and allows inbound connections.`